### PR TITLE
Add transition animation between login epilogue and quick start prompt screens

### PR DESCRIPTION
**Pull request by leandroalonso**
Original PR: #17778

See #17647

To test:

- Fresh install the app
- Login to your WordPress account
- Select personal blog from list
- Observe transition animation to the quick start prompt

## Preview

https://user-images.githubusercontent.com/2454408/149838404-bfe799b9-7b8b-4df7-89c2-1d8f1a3d9d98.mp4

https://user-images.githubusercontent.com/2454408/149838599-2e354af1-4485-42ff-a34c-c9f21285e0a0.mp4

## Regression Notes

1. Potential unintended areas of impact
Margins for the login and sign up screens from the refactor

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Checked the screens on both an iPhone and iPad to verify the margins are still correct.

3. What automated tests I added (or what prevented me from doing so)
None. Unfortunately, testing animations is rather difficult and doesn't add much value.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by leandroalonso** [comment]
@frosty could I bug you to review this one, sir? Thanks!

File: WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift [hunk 1]
```diff
@@ -88,11 +98,18 @@ final class QuickStartPromptViewController: UIViewController {
...
 
     private func setup() {
+        setupScrollViewMargins()
         setupSiteInfoViews()
         setupPromptInfoViews()
         setupButtons()
     }
 
+    private func setupScrollViewMargins() {
+        let margin = view.getHorizontalMargin() + 20.0
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift]: Nitpick, could we move the margin into a constant somewhere?

File: WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift [hunk 2]
```diff
@@ -10,12 +10,17 @@ final class QuickStartPromptViewController: UIViewController {
...
 
     /// Buttons
-    @IBOutlet private weak var showMeAroundButton: FancyButton!
-    @IBOutlet private weak var noThanksButton: FancyButton!
+    @IBOutlet private(set) weak var showMeAroundButton: FancyButton!
+    @IBOutlet private(set) weak var noThanksButton: FancyButton!
+
+    /// Constraints
+    @IBOutlet private(set) weak var scrollViewTopVerticalConstraint: NSLayoutConstraint!
+    @IBOutlet weak var scrollViewLeadingConstraint: NSLayoutConstraint!
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift]: I think these two constraints can be private, as they don't appear to be used elsewhere.

**Review by frosty — APPROVED** [review]
Works great for me! Just two minor comments, otherwise :shipit:

File: WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift (hunk 1 above)
Comment (leandroalonso) [reply WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift]: Done in 2852603ae6866de5d605e74fca4c0463850c4039

File: WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift (hunk 2 above)
Comment (leandroalonso) [reply WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift]: Done in 2852603ae6866de5d605e74fca4c0463850c4039

**Comment by leandroalonso** [comment]
Thanks @frosty!

**Pull request merged**